apps/files/src/components/FilesListVirtual.vue

```diff
@@ -73,17 +73,18 @@
 </template>
 
 <script lang="ts">
-import type { Node as NcNode } from '@nextcloud/files'
+import { getFileActions, Node as NcNode, getFileListHeaders, Folder, View, Permission } from '@nextcloud/files'
```

Comment:
type only imports should be kept

```suggestion
import type { Node as NcNode } from '@nextcloud/files'
```

Reply:
Stupid auto format... thanks